zcbor/zcbor.py

```diff
@@ -2778,25 +2788,33 @@ def parse_args():
     if not args.no_prelude:
         args.cddl.append(open(PRELUDE_path, 'r'))
 
-    if hasattr(args, "decode") and (args.decode == args.encode):
-        parser.error("Please specify exactly one of --decode or --encode.")
+    if hasattr(args, "decode") and not args.decode and not args.encode:
```

Comment:
nit: Would

```suggestion
    if args.decode is None and args.encode is None:
```
work?

Reply:
Sadly no, it raises an exception.